packages/financial-templates-lib/src/price-feed/CreatePriceFeed.js

```diff
@@ -90,6 +91,29 @@ async function createPriceFeed(logger, web3, networker, getTime, config) {
       config.poolDecimals,
       config.priceFeedDecimals // This defaults to 18 unless supplied by user
     );
+  } else if (config.type === "defipulsetvl") {
+    const requiredFields = ["lookback", "minTimeBetweenUpdates"];
+
+    if (isMissingField(config, config.apiKey, requiredFields, logger)) {
+      return null;
+    }
+
+    logger.debug({
+      at: "createPriceFeed",
+      message: "Creating DefiPulseTotalPriceFeed",
+      config
+    });
+
+    return new DefiPulseTotalPriceFeed(
+      logger,
+      web3,
+      config.apiKey,
+      config.lookback,
+      networker,
+      getTime,
+      config.minTimeBetweenUpdates,
+      config.decimals
```

Comment:
nit: for consistency please keep this as
```suggestion
      config.priceFeedDecimals
```